src/unix/stream.c

```diff
@@ -905,10 +912,13 @@ static void uv__write(uv_stream_t* stream) {
       req->send_handle = NULL;
       if (uv__write_req_update(stream, req, n)) {
         uv__write_req_finish(req);
-        return;  /* TODO(bnoordhuis) Start trying to write the next request. */
+        if (cnt-- > 0) {
+          continue; /* Start trying to write the next request. */
+        }
+        return;
       }
     } else if (n != UV_EAGAIN)
-      break;
+      goto error;
```

Comment:
Why this change?

Reply:
I think labeling `error` to the last few lines of this function just makes thing more clear. The original function mixs `return` and `break` in a large loop body, one has to tell the semantic difference between these two control statements.